src/screens/Home/Home.js

```diff
@@ -58,44 +86,110 @@ export function Home({ navigation }) {
   }
 
   const Movie = ({ item }) => (
-    <View>
+    <View style={styles.movieListContainer}>
       <TouchableHighlight accessibilityRole="button" onPress={() => goDetails(item)}>
-        <View style={styles.item} accessibilityIgnoresInvertColors={true}>
-          <View style={styles.avatarContainer}>
-            <Image style={styles.avatar} source={{ uri: `${IMAGE_URL + item.backdropPath}` }} />
-          </View>
-          <Text style={styles.name}>{item.originalTitle}</Text>
-        </View>
+        <Image
+          style={styles.avatar}
+          accessibilityIgnoresInvertColors={true}
+          source={{ uri: `${IMAGE_URL + item.backdropPath}` }}
+        />
       </TouchableHighlight>
       <TouchableWithoutFeedback
         accessibilityRole="button"
-        onPress={() => addMovieFavourites(item)}
+        onPress={() => addMovieFavorites(item)}
         accessibilityIgnoresInvertColors={true}
       >
-        <Image source={addIcon} />
+        <Image style={styles.iconAdd} source={addFavoriteMovieIcon} />
       </TouchableWithoutFeedback>
     </View>
   );
 
-  const headerComponent = () => <Text style={styles.listHeadLine}>{MOVIES}</Text>;
+  const MovieFavorites = ({ item }) => (
+    <View style={styles.movieListContainer}>
+      <TouchableHighlight accessibilityRole="button" onPress={() => goDetails(item)}>
+        <Image
+          style={styles.avatar}
+          accessibilityIgnoresInvertColors={true}
+          source={{ uri: `${IMAGE_URL + item.backdropPath}` }}
+        />
+      </TouchableHighlight>
+      <TouchableWithoutFeedback
+        accessibilityRole="button"
+        onPress={() => removeMovieFromFavorites(item)}
+        accessibilityIgnoresInvertColors={true}
+      >
+        <Image style={styles.iconAdd} source={addFavoriteMovieIcon} />
+      </TouchableWithoutFeedback>
+    </View>
+  );
```

Comment:
I believe that `Movie` and `MovieFavorites` can be moved to their own file instead of combining them with `Home.js`.
This happens because these are different components. Also, move their style objects to a new stylesheet file.